stream-chat-android/src/main/java/com/getstream/sdk/chat/adapter/MessageListItemDiffCallback.kt

```diff
@@ -47,7 +47,7 @@ internal object MessageListItemDiffCallback : DiffUtil.ItemCallback<MessageListI
                 syncStatus = oldItem.message.syncStatus != newItem.message.syncStatus,
                 deleted = oldItem.message.deletedAt != newItem.message.deletedAt,
                 positions = oldItem.positions != newItem.positions,
-                readBy = oldItem.messageReadBy.map { it.getUserId() } == newItem.messageReadBy.map { it.getUserId() }
+                readBy = oldItem.messageReadBy.map { it.getUserId() } != newItem.messageReadBy.map { it.getUserId() }
```

Comment:
🤨 Nice catch) We should pay more attention to tests

Reply:
Haha, good, hard to notice as here the condition is inverted comparing to `areContentsTheSame`